testutil/testproxy/jsonrpc.go

```diff
@@ -1,24 +1,53 @@
 package testproxy
 
-// JSONRpcError is the error struct for the JSON RPC response
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/pokt-network/shannon-sdk/httpcodec"
+)
+
+// JSONRpcError is the error struct for the JSON RPC response payload.
 type JSONRpcError struct {
 	Code    int32  `json:"code"`
 	Message string `json:"message"`
 }
 
-// JSONRpcErrorReply is the error reply struct for the JSON RPC response
+// JSONRpcErrorReply is the error reply struct for the JSON RPC response payload.
 type JSONRpcErrorReply struct {
 	Id      int32  `json:"id"`
 	Jsonrpc string `json:"jsonrpc"`
 	Error   *JSONRpcError
 }
 
-// prepareJsonRPCResponsePayload prepares a hard-coded JsonRPC payload for a specific response.
-func prepareJsonRPCResponsePayload() []byte {
-	return []byte(`{"jsonrpc":"2.0","id":1,"result":"some result"}`)
+// prepareJsonRPCResponse prepares a hard-coded JsonRPC response.
+func prepareJsonRPCResponse() []byte {
+	bodyBz := []byte(`{"jsonrpc":"2.0","id":1,"result":"some result"}`)
+
+	response := &http.Response{
+		StatusCode: http.StatusOK,
+		Header:     http.Header{},
+		Body:       io.NopCloser(bytes.NewReader(bodyBz)),
+	}
+	response.Header.Set("Content-Type", "application/json")
+
+	responseBz, _ := httpcodec.SerializeHTTPResponse(response)
+	return responseBz
 }
 
-// PrepareJsonRPCRequestPayload prepares a hard-coded JsonRPC payload for a specific request.
-func PrepareJsonRPCRequestPayload() []byte {
-	return []byte(`{"method":"someMethod","id":1,"jsonrpc":"2.0","params":["someParam"]}`)
+// PrepareJsonRPCRequest prepares a hard-coded JsonRPC request.
+func PrepareJsonRPCRequest() []byte {
```

Comment:
1. Ditto naming convention (see above).
2. Why is the above unexported (lower case) but this is exported (i.e. upper case)?